homeassistant/components/mystrom/switch.py

```diff
@@ -102,3 +119,13 @@ async def async_update(self) -> None:
             if self._available:
                 self._available = False
                 _LOGGER.error("No route to myStrom plug")
+
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return the device info for the light entity."""
+        return DeviceInfo(
+            identifiers={(DOMAIN, self.plug.mac)},
+            name=self.name,
+            manufacturer=ATTR_MANUFACTURER,
+            sw_version=self.plug.firmware,
+        )
```

Comment:
This can be added as `self._attr_device_info` in `__init__` method.

Reply:
done